apps/style/spritelab/style.scss

```diff
@@ -1 +1,9 @@
 @import "../gamelab/style.scss";
+
+button.arrow:disabled {
+  pointer-events: none;
+}
+
+button#pauseButton:disabled {
+  pointer-events: none;
+}
```

Comment:
i don't think we should do this here, but if it's useful in the future, Sass has an [`@extend` rule](https://sass-lang.com/documentation/at-rules/extend)

Reply:
we could also do something like:

```scss
button {
  &.arrow:disabled,
  &#pauseButton:disabled {
    pointer-events: none;
  }
}
```